encoder-brave/README.md

```diff
@@ -0,0 +1,8 @@
+# encoder-brave
+
+This encodes brave spans into OTLP proto format.
+
+```java
+// connect the sender to the correct encoding
```

Comment:
`correct encoder`

Or maybe

`Use OTLP encoder when sending to an OTLP backend`

Reply:
Fixed in https://github.com/openzipkin-contrib/zipkin-otel/pull/9/commits/b66e3a659186872a4573d618434aa250c643ce4a